pinc/MARCRecord.inc

```diff
@@ -7,11 +7,263 @@ class InvalidMARCRecord extends Exception
+//    ["(3,650)(3, 0)(3,a)", "Slavery"],
+//    ["(3,650)(3, 0)(3,z)", "United States."],
+//    ["(3,650)"],
+//    ["(3,650)(3, 0)"],
+//    ["(3,650)(3, 0)(3,a)", "Speeches, addresses, etc., American."],
+//    ["(3,655)"],
+//    ["(3,655)(3, 7)"],
+//    ["(3,655)(3, 7)(3,a)", "Speeches."],
+//    ["(3,655)(3, 7)(3,2)", "lcgft"],
+//    ["(3,655)"],
+//    ["(3,655)(3, 7)"],
+//    ["(3,655)(3, 7)(3,a)", "Literary readings."],
+//    ["(3,655)(3, 7)(3,2)", "lcgft"],
+//    ["(3,655)"],
+//    ["(3,655)(3, 7)"],
+//    ["(3,655)(3, 7)(3,a)", "Sound recordings."],
+//    ["(3,655)(3, 7)(3,2)", "lcgft"],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Stewart, Maria W.,"],
+//    ["(3,700)(3,1 )(3,d)", "1803-1879,"],
+//    ["(3,700)(3,1 )(3,e)", "author."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Garnet, Henry Highland,"],
+//    ["(3,700)(3,1 )(3,d)", "1815-1882,"],
+//    ["(3,700)(3,1 )(3,e)", "author."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Douglass, Frederick,"],
+//    ["(3,700)(3,1 )(3,d)", "1818-1895,"],
+//    ["(3,700)(3,1 )(3,e)", "author."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Washington, Booker T.,"],
+//    ["(3,700)(3,1 )(3,d)", "1856-1915,"],
+//    ["(3,700)(3,1 )(3,e)", "author."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "McNeil, Claudia,"],
+//    ["(3,700)(3,1 )(3,e)", "speaker."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Matlock, Norman,"],
+//    ["(3,700)(3,1 )(3,e)", "speaker."],
+//    ["(3,700)"],
+//    ["(3,700)(3,1 )"],
+//    ["(3,700)(3,1 )(3,a)", "Graham, John,"],
+//    ["(3,700)(3,1 )(3,d)", "1926-2007,"],
+//    ["(3,700)(3,1 )(3,e)", "editor."],
+//  ]
+
 class MARCRecord
 {
     // MARC record in the yaz_search 'array' format
     private $record = [];
-    private $literary_form_array = ["a" => "Art", "b" => "Biography", 3 => "Comedy", "c" => "Comic Strip", 4 => "Cooking", "d" => "Drama", "e" => "Essay", 1 => "Fiction", "g" => "Geography", 5 => "Historical", 6 => "History", 7 => "Humor", "i" => "Letter", "l" => "Linguistics", 8 => "Math", 9 => "Medicine", "m" => "Mixed Form", "v" => 'Music', 0 => "Non-Fiction", "f" => "Novel", "y" => "Periodical", "p" => "Poetry", "r" => "Romance", "z" => "Science", "h" => "Satire", "j" => "Short Story", "s" => "Speech", "u" => "Unknown", "|" => "Unknown"];
+    private $literary_form_array = [
+        "a" => "Art",
+        "b" => "Biography",
+        3 => "Comedy",
+        "c" => "Comic Strip",
+        4 => "Cooking",
+        "d" => "Drama",
+        "e" => "Essay",
+        1 => "Fiction",
+        "g" => "Geography",
+        5 => "Historical",
+        6 => "History",
+        7 => "Humor",
+        "i" => "Letter",
+        "l" => "Linguistics",
+        8 => "Math",
+        9 => "Medicine",
+        "m" => "Mixed Form",
+        "v" => 'Music',
```

Comment:
Should this be `"`Music`"` to match the rest?

Reply:
Done!